public/locales/en/translation.json

```diff
@@ -18,6 +18,10 @@
     "Lot": "Lot",
     "Date": "Date",
     "Vaccinator": "Vaccinator",
+    "Lab Test": "Lab Test",
+    "Lab Performer": "Lab Performer",
+    "Ref. Range": "Ref. Range",
```

Comment:
I had originally abbreviated it because its length was causing it to carry over into where the value is displayed, and I didn't want to make the styling inconsistent with the rest of the fields. But I agree, the abbreviation might not be intuitive. I tried moving "range" to a new line, but it makes it look like it's own field, which also could cause confusion and doesn't look too nice stylistically. Open to suggestions on what you think the best handling for this is.

Reply:
In https://github.com/dvci/shc-web-verifier/pull/77/commits/357c65de50ef272c444fc8b4feb3151c2d8d5404, I increased the `paddingLeft` of the `gridItem` style so that the "Reference Range" grid label would not overlap with the reference range values displayed. Let me know what you think.

Reply:
This was slightly modified in https://github.com/dvci/shc-web-verifier/pull/77/commits/fd1aa80c6f0aa89dcc28d2fa624c3df354ecc673 so that all the grid labels are aligned.